<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_gestaousuarios</class>
 <widget class="QDialog" name="frm_gestaousuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestão de Usuários</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>671</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_gestaocolaboradores">
    <attribute name="title">
     <string>Novo Usuário</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>403</x>
       <y>424</y>
       <width>257</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QPushButton" name="btn_nv_novo">
        <property name="text">
         <string>Novo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_nv_gravar">
        <property name="text">
         <string>Salvar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>222</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_27">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_26">
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Nome</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_nv_nome"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Usuário</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_nv_usuario"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_28">
          <item>
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Senha</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_nv_senha">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Tipo de Acesso</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_nv_acesso"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_novocolaborador">
    <attribute name="title">
     <string>Gestão Usuários</string>
    </attribute>
    <widget class="QTableWidget" name="tw_ge_listausuario">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>361</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_8">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>221</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_31">
      <item>
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Filtro</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_ge_filtro"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ge_filtrar">
        <property name="text">
         <string>Filtrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>281</width>
       <height>431</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_32">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_33">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Nome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_ge_nome"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_34">
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Usuário</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_ge_usuario"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_35">
        <item>
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Senha</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_ge_senha">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_36"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_37">
        <item>
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>Tipo de Acesso</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_ge_acesso"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QPushButton" name="btn_ge_salvar">
          <property name="text">
           <string>Salvar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_ge_excluir">
          <property name="text">
           <string>Excluir</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Filtrar por:</string>
     </property>
     <widget class="QWidget" name="layoutWidget_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>114</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_38">
       <item>
        <widget class="QRadioButton" name="rb_ge_id_usuario">
         <property name="text">
          <string>ID Usuario.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_ge_nome_usuario">
         <property name="text">
          <string>Nome</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
